graphics/ogre/Portfile

```diff
@@ -21,83 +21,67 @@ long_description    OGRE (Object-Oriented Graphics Rendering Engine) is a \
                     library abstracts all the details of using the \
                     underlying system libraries like Direct3D and \
                     OpenGL and provides an interface based on world \
-                    objects and other intuitive classes. \
-                    NOTE: \
-                    This version is so far the only version which produces \
-                    normal non-bundled executables that can run on any \
-                    64-bit MacOS X machine without depending on MacPorts or \
-                    any other libraries. This version is also the only \
-                    version that allows building fully functioning CMake \
-                    OGRE projects on MacOS X without using Xcode.
+                    objects and other intuitive classes.
 
 homepage            https://www.ogre3d.org/
 platforms           darwin
 supported_archs     x86_64
-master_sites        sourceforge:project/ogre/ogre/${branch}
-checksums           rmd160  fea467f35aaf0fd5926573de4f0348ed44191893 \
-                    sha1    41acccfbbf7297c91cda78a0ce8a053e56505f04
-distname            ogre_src_v[strsed ${version} {g/\./-/}]
-use_bzip2           yes
 
-boost.version       1.71
+checksums           rmd160  52c0dd24b8ec427d35a97c2b73746298653ce8db \
+                    sha1    272345936aa286f848bb8aa1691521b34044ba53 \
+                    sha256  8d31a79d37f74e5af25e56a0a70928079e771f7cea50ef041e04bf235c3f701a \
+                    size    91796574
 
-depends_build-append port:doxygen path:bin/dot:graphviz port:pkgconfig
+use_xcode           yes
 
-depends_lib-append  port:libzzip port:zlib port:bzip2 port:freeimage \
-                    port:freetype port:ois
+cmake.build_type    Release
+compiler.cxx_standard \
+                    2011
+
+depends_build-append \
+                    bin:dot:graphviz \
+                    port:doxygen \
+                    port:pkgconfig
+
+depends_lib-append  port:bzip2 \
+                    port:assimp \
+                    port:freeimage \
+                    port:freetype \
+                    port:glslang \
+                    port:ilmbase \
```

Comment:
this will never be used if freeimage is enabled

Reply:
Thanks!